backend/.env

```
@@ -0,0 +1,127 @@
+
+
+# =============================================================================
+# STOCK SCANNER ENVIRONMENT CONFIGURATION (DUAL DB: default + stocks)
+# =============================================================================
+DJANGO_SETTINGS_MODULE=stockscanner_django.settings_production
+DJANGO_DEBUG=False
+ENVIRONMENT=production
+
+# Hosts / Frontend
+DJANGO_SECRET_KEY="((#cx+mb@f-(8x*p@9mfnanqe%ha1@6-b%w)q##v@)lanop"
+DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1,retailtradescannet.com,api.retailtradescannet.com,tradescanpro.com,www.tradescanpro.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DJANGO_ALLOWED_HOSTS key should go before the DJANGO_SECRET_KEY key

(UnorderedKey)